evap/static/ts/src/confirmation-modal.ts

```diff
@@ -68,6 +68,10 @@ export class ConfirmationModal extends HTMLElement {
         }
     };
 
+    show = () => {
```

Comment:
There is another occurrence of showModal in staff_evaluation_operation that should be changed to use this new .show function, as mentioned here: https://github.com/e-valuation/EvaP/pull/2469#discussion_r2203501978